tests/contenttypes_tests/test_management.py

```diff
@@ -12,8 +12,6 @@
 
 @modify_settings(INSTALLED_APPS={'append': ['no_models']})
 class RemoveStaleContentTypesTests(TestCase):
-    # Speed up tests by avoiding retrieving ContentTypes for all test apps.
-    available_apps = ['contenttypes_tests', 'no_models', 'django.contrib.contenttypes']
 
```

Comment:
we need to retrieve all test apps to prove that our code change will only remove apps that are no longer in the installed apps list.